src/pages/ProfilePassword&SecurityPage.jsx

```diff
@@ -0,0 +1,85 @@
+import { useState } from "react";
+import ButtonProfile from "../shared/ButtonProfile";
+import ProfileHeader from "../shared/ProfileHeader";
+
+const ProfilePasswordSecurityPage = () => {
```

Comment:
Please store all these pages inside a folder in the pages folder in src, just for organize everything, also rename the profilePassword&SecurityPage to profilePasswordSecurityPage or profileSecurityPage, just remove the sign.